src/workspace-data/ImportStatus.js

```diff
@@ -40,6 +41,9 @@ function ImportStatusItem({ job: { targetWorkspace, jobId }, onDone }) {
   const checkForCompletion = async ({ namespace, name }, jobId) => {
     const fetchImportStatus = async () => {
       try {
+        if (isAzureWorkspace(targetWorkspace)) {
+          return await Ajax(signal).WorkspaceData.getJobStatus(proxyUrl, jobId);
+        }
         return await Ajax(signal).Workspaces.workspace(namespace, name).getImportJobStatus(jobId);
       } catch (error) {
         // Ignore 404; We're probably asking for status before the status endpoint knows about the job
```

Comment:
I know you didn't change this line, but this block ignoring 404 seems a bit suspect.  Especially if we also think we can safely apply it to WDS.  At the least I think it might be a good idea to bubble up 404 when talking to WDS.